### ISSUE TITLE
pip installation attr vs attrs

### ISSUE THREAD

**Issue by agoscinski** [Luthaf/rascaline#2]
`pip install .` 
gives me output
```
  ERROR: Command errored out with exit status 1:
   command: /opt/anaconda3/bin/python /opt/anaconda3/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-jqgeiwja/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools >=44' 'wheel >=0.36' 'attr >=20'
       cwd: None
  Complete output (6 lines):
  Collecting setuptools>=44
    Using cached setuptools-53.0.0-py3-none-any.whl (784 kB)
  Collecting wheel>=0.36
    Using cached wheel-0.36.2-py2.py3-none-any.whl (35 kB)
  ERROR: Could not find a version that satisfies the requirement attr>=20 (from versions: 0.1.0, 0.2.0, 0.3.0, 0.3.1)
  ERROR: No matching distribution found for attr>=20
  ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/anaconda3/bin/python /opt/anaconda3/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-jqgeiwja/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools >=44' 'wheel >=0.36' 'attr >=20' Check the logs for full command output.
```
I switched in the pyproject.toml
https://github.com/Luthaf/rascaline/blob/a928ba16e788e977a5c8934ec923ff4cc946e10c/pyproject.toml#L5
attr to attrs

**Comment by Luthaf** [comment]
Looks like there is no need to depend on attr here, `attr: rascaline.__version__` is provided in setup.cfg by setuptools. I'll remove the dependency =)

**Comment by Luthaf** [comment]
Fixed in ccfd3a3e914696278ad52cbf71a54a91b2db2779
